api/src/pcapi/core/finance/models.py

```diff
@@ -719,3 +725,100 @@ class PaymentMessage(Base, Model):
+    FRAUD = "FRAUD"
+
+
+class Incident(Base, Model):
+    id: int = sqla.Column(sqla.BigInteger, primary_key=True, autoincrement=True)
+    type: IncidentType = sqla.Column(
+        sqla.Enum(IncidentType, native_enum=False, create_contraint=False),
+        nullable=False,
+    )
+    status: IncidentStatus = sqla.Column(
+        sqla.Enum(IncidentStatus, native_enum=False, create_constraint=False),
+        nullable=False,
+        server_default=IncidentStatus.CREATED.value,
+        default=IncidentStatus.CREATED,
+    )
+
+    userId: int = sqla.Column(sqla.BigInteger, sqla.ForeignKey("user.id"), index=True, nullable=False)
+    user: sqla_orm.Mapped["users_models.User"] = sqla_orm.relationship(
+        "User", foreign_keys=[userId], backref="incidents"
+    )
+
+    # bookings: sqla_orm.Mapped[list["bookings_models.Booking"]] = sqla_orm.relationship(
+    #     "Booking", back_populates="incident"
+    # )
+    #
+    venueId = sqla.Column(sqla.BigInteger, sqla.ForeignKey("venue.id"), nullable=True)
+    venue: sqla_orm.Mapped["offerers_models.Venue | None"] = sqla_orm.relationship("Venue", foreign_keys=[venueId])
+
+    details: dict | None = sqla.Column(
+        sqla_mutable.MutableDict.as_mutable(sqla_psql.JSONB), nullable=True, default={}, server_default="{}"
+    )
+
+    @property
+    def invoiced_pricing(self) -> Pricing | None:
+        processed_pricings = [pricing for pricing in self.pricings if pricing.status == PricingStatus.INVOICED]
+
+        if not processed_pricings:
+            return None
+
+        pricings = sorted(processed_pricings, key=attrgetter("creationDate"), reverse=True)
+
+        if pricings:
+            return pricings[0]
+
+        return None
+
+    @property
+    def regulation_cashflow_batch(self) -> CashflowBatch | None:
+        if not self.pricing:
+            return None
+
+        cashflow = self.pricing.cashflow
+        if not cashflow:
+            return None
+
+        return cashflow.batch
+
+
+class BookingFinanceIncident(Base, Model):
+    id: int = sqla.Column(sqla.BigInteger, primary_key=True, autoincrement=True)
+
+    bookingId = sqla.Column(sqla.BigInteger, sqla.ForeignKey("booking.id"), nullable=True)
+    booking: sqla_orm.Mapped["bookings_models.Booking | None"] = sqla_orm.relationship(
+        "Booking", foreign_keys=[bookingId]
+    )
+
+    collectiveBookingId = sqla.Column(
+        sqla.BigInteger, sqla.ForeignKey("collective_booking.id"), index=True, nullable=True
+    )
+    collectiveBooking: "educational_models.CollectiveBooking" = sqla_orm.relationship(
+        "CollectiveBooking",
+        foreign_keys=[collectiveBookingId],
+    )
+
+    incidentId = sqla.Column(sqla.BigInteger, sqla.ForeignKey("incident.id"), nullable=True)
+    incident: sqla_orm.Mapped["Incident | None"] = sqla_orm.relationship("Incident", foreign_keys=[incidentId])
+
+    newTotalAmount: decimal.Decimal = sqla.Column(sqla.Numeric(10, 2), nullable=False)
```

Comment:
Les autres colonnes contenant des montants dans ce module (*) sont des entiers (exprimant une valeur en centimes). Par souci d'homogénéité, il faudrait en faire autant ici.

(*) ok, toutes sauf `CustomReimbursementRule.amount`.